app/templates/items/new.hbs

```diff
@@ -136,7 +136,7 @@
 
     <div class="{{if isMobileApp 'small-7 large-8' 'small-9'}} columns">
       {{#link-to "select_location"}}
-        <div class="{{if (is-or invalidLocation invalidScanResult) 'ember-view form__control form__control--error'}}">
+        <div class="{{if (is-or invalidLocation invalidScanResult) 'ember-view form__control form__control--error'}}" id="inventory_location">
```

Comment:
Can we assign ID to input rather then to div?

Reply:
If in case value is not present in input we can fetch error value from child div.